### ISSUE TITLE
weekStart option has no effects

### ISSUE THREAD

**Issue by mariogl** [koleary94/Angular-2-Datepicker#89]
I'm setting weekStart option to 1, but week always start on Sunday.

In the component, I can see that class Calendar is instantiated without passing the _firstWeekDay_ parameter, so the object is always created with its _firstWeekDay_ property setted to 0.

If this omission is intentional, then the option weekStart shouldn't be documented in README.md, as is now.

**Comment by joangilabert94** [comment]
The file src/datepicker.component.ts in github it is different than the npm. In npm weekStart and month not work.

**Comment by LucaPierfederici** [comment]
Yes I noticed the same thing. It would be great if _weekStart_, _month_, and _dayNames_ will be supported soon in npm version @koleary94
